trieste/acquisition/function.py

```diff
@@ -176,24 +187,27 @@ def prepare_acquisition_function(
         fmean, fvar = model.predict(query_points)
         fsd = tf.math.sqrt(fvar)
 
-        def probf(x: tf.Tensor) -> tf.Tensor :  # Build empirical CDF
+        def probf(y: tf.Tensor) -> tf.Tensor :  # Build empirical CDF for Pr(y*^hat<y)
             unit_normal = tfp.distributions.Normal(tf.cast(0, fmean.dtype), tf.cast(1, fmean.dtype))
-            log_cdf = unit_normal.log_cdf(- (x - fmean) / fsd)
-            return tf.exp(tf.reduce_sum(log_cdf, axis=0))
+            log_cdf = unit_normal.log_cdf(- (y - fmean) / fsd)
+            return 1 - tf.exp(tf.reduce_sum(log_cdf, axis=0))
```

Comment:
i'm finding it difficult to wrap my head around the signs here. Could you explain them?